next-tavla/src/Admin/scenarios/Landing/styles.module.css

```diff
@@ -0,0 +1,15 @@
+.container {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    gap: 3rem;
+}
+
+.header {
+    text-align: center;
+}
+
+.createButton {
+    max-width: 20rem;
+    margin: 0 auto;
+}
```

Comment:
Dette virker litt for komplisert for det det skal oppnå og kan antagelig forenkles.